exir/tests/test_memory_format_ops_pass.py

```diff
@@ -376,6 +377,74 @@ def call_operator(self, op, args, kwargs, meta):
         self.assertTrue(is_contiguous_dim_order(actual))
         self.assertTrue(is_contiguous_dim_order(expected))
 
+    def test_op_clone_replacement_channels_last_survives(self):
```

Comment:
lets move test to [test_remove_clone_ops.py](https://github.com/pytorch/executorch/blob/main/backends/transforms/test/test_remove_clone_ops.py)